libvtkfortran/vtkfortran.cpp

```diff
@@ -773,25 +769,7 @@ extern "C" {
   */
   void vtkpclose(int *rank, int *npartitions){
 
-#ifdef HAVE_MPI
-    // Interleaving is experimental - play at your own risk
-#define INTERLEAVE_IO_TRESHOLD 64000
-#define CORES_PER_NODE 8
-    if(*npartitions>INTERLEAVE_IO_TRESHOLD){
-      int nwrites = (int)(sqrt(*npartitions)+0.5);
-      
-      for(int lrank=0; lrank<nwrites; lrank++){
-        if((*rank)%nwrites==lrank){
-          _vtkpclose_nointerleave(rank, npartitions);
-        }
-        MPI_Barrier(MPI_COMM_WORLD);
-      }
-    }else{
-#endif
-      _vtkpclose_nointerleave(rank, npartitions);
-#ifdef HAVE_MPI
-    }
-#endif
+    _vtkpclose_nointerleave(rank, npartitions);
```

Comment:
So no interleaving/MPI in libvtkfortran?

Reply:
It seemed very dodgy and only seems to kick in on npartiions>64000 (!), so I thought it'd be safe to remove.

Reply:
But as to how this relates to this PR, it means there's no explicit MPI calls in libvtkfortran anymore which means I don't have to introduce this mpi header business for libvtkfortran.  DQMOM is indeed unrelated, so you're right I should have done that in a separate merge.